stk/backend/__init__.py

```diff
@@ -1 +1 @@
-from stk.backend.sputnik import dsd
+from stk.backend.sputnik import dsd
```

Comment:
I think we can delete this line? Looks like you might've just touched this file by accident.